app/presenters/navigation_presenter.rb

```diff
@@ -44,6 +44,10 @@ def nav_footer
       {
         title: 'Explore Commitments',
         url: commitments_path
+      },
+       {
+        title: 'CONTACT US: nature.commitments@unep-wcmc.org',
+        url: 'not a link'
```

Comment:
as this is not really a link I would move this out of this array, either creating a new method or refactoring this a bit to include both links and non-links without having to apply that logic on the .erb file